Assets/Scripts/PS4 Controller.cs

```diff
@@ -0,0 +1,71 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class PS4Controller : MonoBehaviour
+{
+    //variables
+    float movementSpeed = 5f;
```

Comment:
I always find it useful to include private here, someone who is just beginning mightn't be aware that variables are private by default. Will approve was just adding my preference for this